compiler/src/main/kotlin/io/github/apl_cornell/viaduct/codegeneration/BackendCodeGenerator.kt

```diff
@@ -229,6 +238,60 @@
         }
     }
 
+    private fun simpleStatement(protocol: Protocol, stmt: SimpleStatementNode): CodeBlock {
+        return when (stmt) {
+            is LetNode -> let(protocol, stmt)
+
+            is DeclarationNode -> declaration(protocol, stmt)
+
+            is UpdateNode -> codeGenerator.update(protocol, stmt)
+
+            is OutParameterInitializationNode -> outParameterInitialization()
+
+            is OutputNode -> CodeBlock.of(
+                "runtime.output(%T(%L))",
+                typeAnalysis.type(stmt.message).valueClass,
+                codeGenerator.exp(protocol, stmt.message)
+            )
+        }
+    }
+
+    private fun let(protocol: Protocol, stmt: LetNode): CodeBlock =
```

Comment:
These (`let`, `declaration`, `outParameterInitialization` etc.) are short enough that you should just inline them.